src/Tasks.UnitTests/CombinePath_Tests.cs

```diff
@@ -203,6 +203,7 @@ public void BlankPath()
         /// </summary>
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // No invalid characters on Unix
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp)]
```

Comment:
Why are the `SkipOnTargetFramework` attributes needed? Shouldn't .net core on Windows fail with the same IO constraints?